roles/distributive/tasks/main.yml

```diff
@@ -8,10 +8,34 @@
     dest: /etc/sudoers.d/consul-docker-auth
     validate: visudo -c -f "%s"
   tags:
+    - bootstrap
     - consul
     - distributive
     - docker
 
+# Needed for upgrade from Mantl <1.0
+- name: remove symlink to deprecated distributive checklist
+  become: yes
+  file:
+    state: absent
+    path: /etc/distributive.d/{{ current_role }}.json
+  tags:
+    - bootstrap
+    - distributive
+
+# Needed for upgrade from Mantl <1.0
+- name: de-register deprecated distributive tests with consul
+  become: yes
+  file:
+    path: "/etc/consul/distributive-{{ current_role }}-check.json"
+    state: absent
+  notify:
+    - reload consul
+  tags:
+    - bootstrap
+    - consul
+    - distributive
+
```

Comment:
I tried upgrading a 1.0.3 cluster with this and I still end up with the distributive checks in the consul directory and the distributive directory:

```
$ sudo ls -al /etc/consul
total 72
drwxrwx---.  3 consul consul 4096 Mar 17 20:32 .
drwxr-xr-x. 95 root   root   8192 Mar 17 20:22 ..
-rw-r--r--.  1 consul consul  163 Mar 17 19:57 acl.json
-rw-r--r--.  1 root   root     57 Mar 17 19:58 agent_acl.json
-rw-rw-r--.  1 consul consul  753 Mar 17 20:21 consul.json
-rw-r--r--.  1 root   root    995 Mar 17 19:59 consul.nginx
-rw-r--r--.  1 root   root     41 Jan 19 14:51 consul-ui.json
-rw-r--r--.  1 consul consul  276 Mar 17 19:52 distributive-common-check.json
-rw-r--r--.  1 root   root    296 Mar 17 20:32 distributive-consul-check.json
-rw-r--r--.  1 root   root    299 Mar 17 20:32 distributive-dnsmasq-check.json
-rw-r--r--.  1 root   root    302 Mar 17 20:32 distributive-logstash-check.json
-rw-r--r--.  1 root   root    299 Mar 17 20:32 distributive-mantlui-check.json
-rw-r--r--.  1 root   root    333 Mar 17 20:32 distributive-zookeeper-check.json
-rw-r--r--.  1 consul consul  176 Mar 17 19:57 docker-service.json
-rw-r--r--.  1 root   root    312 Mar 17 20:02 marathon.json
drwxrwxr-x.  2 consul consul   55 Mar 17 19:57 ssl
-rw-r--r--.  1 root   root    261 Mar 17 20:00 zk-consul.json
```

```
ls -al /etc/distributive.d/
total 36
drwxr-xr-x.  2 root root 4096 Mar 17 20:24 .
drwxr-xr-x. 95 root root 8192 Mar 17 20:22 ..
lrwxrwxrwx.  1 root root   35 Mar 17 19:52 common.json -> /usr/share/distributive/common.json
-rw-rw-r--.  1 root root 1566 Feb  9 21:54 distributive-consul.json
-rw-rw-r--.  1 root root 1336 Mar 14 22:01 distributive-dnsmasq.json
-rw-rw-r--.  1 root root 1178 Mar 11 14:04 distributive-logstash.json
-rw-rw-r--.  1 root root  910 Mar 11 19:28 distributive-mantlui.json
-rw-rw-r--.  1 root root 1213 Feb  9 21:57 distributive-zookeeper.json
```

There were no errors during the ansible run. And all nodes are unhealthy b/c of this after the upgrade. Any idea?